app/main.py

```diff
@@ -1,8 +1,37 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
```

Comment:
The `people` dictionary is defined as a class attribute. This is fine for keeping track of all `Person` instances by name, but be cautious about potential memory leaks if `Person` instances are not properly managed or removed when no longer needed.